### PR TITLE
feat: add linux/arm64 arch

### LINKED ISSUE #65: Publish ARM images
## Description

Newer MacBooks (and other laptops) are now using ARM processors, and so these base images don't support that by default - though they can with emulation but this is considerably slower than using an ARM images.
Additionally, many robot target platforms use ARM, so it would be useful to provide CI environments matching the target architecture.

We should create and push ARM images in parallel to the x86 ones that we are currently publishing.

We'd like to create ARM 

## Related Issues

N/A

## Completion Criteria

ARM versions of all images are pushed as part of standard build.

## Implementation Notes / Suggestions

I don't think there are GitHub Actions ARM runners available out of the box. Could possibly just trust a slower QEMU build to get the job done.

## Testing Notes / Suggestions

N/A

### PR DESCRIPTION
**Pull request by wep21**
Closes #65

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -141,7 +141,7 @@ jobs:
       uses: docker/build-push-action@v5
       with:
         context: .
-        platforms: linux/amd64
+        platforms: linux/amd64,linux/arm64
```
Comment (christophebedard) [inline .github/workflows/build.yml:144]: The base Ubuntu image should probably also have this.

Comment (wep21) [reply .github/workflows/build.yml:144]: @christophebedard also added arm64 into base https://github.com/ros-tooling/setup-ros-docker/pull/70/commits/d463c20e6065cfba7eb9500b83649692670a739f.

**Review by christophebedard — APPROVED** [review]
Looks good. From the CI output, building the linux/arm64 seems to be working fine.

**Comment by wep21** [comment]
@christophebedard Thank you fro your review. Is it okay to go ahead to merge?

**Comment by christophebedard** [comment]
Yep! The Noble jobs are failing, but that's expected.

**Comment by christophebedard** [comment]
Thanks for the PR!

**Pull request merged**